truncate_client/src/game.rs

```diff
@@ -29,6 +29,30 @@ pub fn render(client: &mut GameClient, ui: &mut egui::Ui) {
         frame_history: _,
     } = client;
 
+    // TODO: The next four blocks are due to the fact that we're
+    // using futures::channel::mpsc on wasm, and tokio::sync::mpsc
+    // on native. We should probably look at simply _always_ using
+    // futures::channel::mpsc so that we don't have to do all of this.
```

Comment:
Because I don't have a tokio stack in the wasm, so there were a couple of libraries I had to swap out to get things working there and I haven't spent the time to check if they work on the native impl. Might look @ that this weekend before merging.

Theoretically you get bonuses when using tokio types with the tokio runtime, but I don't know if they're relevant to standard `mpsc` channels, it's mainly for files/io.